dot/network/service.go

```diff
@@ -368,6 +375,10 @@ func (s *Service) updateMetrics() {
 	}
 }
 
+func getProcessStartTime() float64 {
+	return float64(time.Now().Unix())
```

Comment:
nit remove this function and set the gauge directly with

```go
processStartTimeGauge.Set(float64(time.Now().Unix()))
```

Reply:
Updated.